app/assets/javascripts/index/new_note.js

```diff
@@ -35,11 +35,13 @@ OSM.NewNote = function (map) {
     OSM.router.route("/note/new");
   });
 
-  function createNote(marker, form, url) {
-    var location = marker.getLatLng().wrap();
+  function createNote(form, url) {
+    if (!newNoteMarker) return;
```

Comment:
Why do we now need to check this? As far as I can see nothing has changed and if can be null now then it always could be?

Reply:
Made some changes to be sure that:
- `addNewNoteMarker` is called from `page.load`
- `removeNewNoteMarker` is called only from `page.unload`
- `createNote` doesn't access the marker